<?xml version='1.0' encoding='UTF-8'?>
<root>
 <tabbed_widget name="Main Window" parent="main_window">
  <plotmatrix tab_name="plot" columns="1" rows="2">
   <plot row="0" col="0">
    <range top="0.164837" bottom="-0.028195" left="0.000000" right="97.859038"/>
    <limitY/>
    <curve name="/cam_pose_in_world_kf/input_position/x" custom_transform="noTransform" B="0" R="0" G="128"/>
    <curve name="/cam_pose_in_world_kf/position_kf/x" custom_transform="noTransform" B="255" R="0" G="0"/>
    <transform value="noTransform"/>
   </plot>
   <plot row="1" col="0">
    <range top="0.203784" bottom="-0.277295" left="0.000000" right="97.859038"/>
    <limitY/>
    <curve name="/cam_pose_in_world_kf/acceleration_kf/x" custom_transform="" B="255" R="255" G="0"/>
    <curve name="/cam_pose_in_world_kf/position_kf/x" custom_transform="" B="0" R="0" G="128"/>
    <curve name="/cam_pose_in_world_kf/velocity_kf/x" custom_transform="" B="0" R="255" G="0"/>
    <transform value=""/>
   </plot>
  </plotmatrix>
  <currentPlotMatrix index="0"/>
 </tabbed_widget>
 <use_relative_time_offset enabled="1"/>
 <Plugins>
  <DataLoad_CSV>
   <default time_axis=""/>
  </DataLoad_CSV>
  <DataLoad_ROS_bags>
   <selected_topics list=""/>
  </DataLoad_ROS_bags>
  <DataLoad_ULog>
   <no_params/>
  </DataLoad_ULog>
  <ROS_Topic_Streamer>
   <selected_topics list="/cam_pose_in_world;/cam_pose_in_world_kf"/>
  </ROS_Topic_Streamer>
  <RosoutPublisherROS/>
  <TopicPublisherROS/>
 </Plugins>
 <previouslyLoadedStreamer name="ROS_Topic_Streamer"/>
 <customMathEquations/>
 <snippets/>
</root>

